contracts/interfaces/Accounts.sol

```diff
@@ -5,5 +5,19 @@ pragma solidity ^0.8.18;
  * @title ISafe Declares the functions that are called on a Safe by Safe protocol.
  */
 interface ISafe {
+    function setModule(address _module) external;
```

Comment:
This is not part of the Safe interface and should be removed from this interface